csharp/ql/lib/semmle/code/csharp/dataflow/internal/FlowSummaryImpl.qll

```diff
@@ -219,6 +219,11 @@ module Public {
      */
     pragma[nomagic]
     predicate clearsContent(ParameterPosition pos, ContentSet content) { none() }
+
+    /**
+     * Gets whether the summary is auto generated or not.
+     */
+    boolean isAutoGenerated() { result = false }
```

Comment:
It is more natural for this to simply be a `predicate isAutoGenerated()`.

Reply:
Check, will fix it :-)